UPGRADE-3.0.md

```diff
@@ -114,6 +114,12 @@ UPGRADE FROM 2.x to 3.0
 
 ### Form
 
+ * The `getBlockPrefix()` method was added to the `FormTypeInterface` in replacement of
+   the `getName()` method which has been has been removed.
+ * The `configureOptions()` method was added to the `FormTypeInterface` in replacement
```

Comment:
It would be good to show the different signature to reflect the change from OptionsResolverInterface to OptionsResolver